controllers/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -718,12 +717,11 @@ var _ = Describe("HyperconvergedController", func() {
 				existingResource.Kind = kubevirtcorev1.KubeVirtGroupVersionKind.Kind // necessary for metrics
 
 				// now, modify KV's node placement
-				seconds3 := int64(3)
 				existingResource.Spec.Infra.NodePlacement.Tolerations = append(hco.Spec.Infra.NodePlacement.Tolerations, corev1.Toleration{
-					Key: "key3", Operator: "operator3", Value: "value3", Effect: "effect3", TolerationSeconds: &seconds3,
+					Key: "key3", Operator: "operator3", Value: "value3", Effect: "effect3", TolerationSeconds: ptr.To[int64](3),
```

Comment:
here since we use it 4 times maybe worth creating the var `seconds3 := ptr.To[int64](3)`?

Reply:
I don't think it saves something. I prefer to not have redundant variables.